Decode/PerfItemValue.cs

```diff
@@ -1624,492 +1972,30 @@ public string GetString()
         }
 
         /// <summary>
-        /// Appends a string representation of this value (suitable for diagnostic use) like
-        /// "Type:Value" or "Type:Value1,Value2". Result is the same as from this.ToString(convertOptions).
-        /// Returns sb.
+        /// Returns a string representation of this value. Result is the same as from
+        /// this.AppendTo(PerfConvertOptions.Default).
         /// </summary>
-        public StringBuilder AppendTo(
```

Comment:
Note: Previously there were three output choices: ToString, non-JSON, and JSON. The ToString implementation came first, and it was quick-and-dirty and focused on "useful to see in a debugger".

Now that the non-JSON version is finished, maintaining a separate ToString version seems unnecessary, so merge the two.
- Delete the ToString version. Re-implement ToString as a call to the non-JSON version.
- The non-JSON version should no longer throw in edge cases like default-initialized Value objects.

The non-JSON version isn't quite as useful for debugging as the original ToString version, but I don't think the added value is worth a whole extra formatter. Better to make improvements to the non-JSON formatter if necessary in the future rather than have a whole separate formatter.